Makefile

```diff
@@ -138,20 +142,52 @@ ifeq ($(NVIDIA),1)
 endif
 
 PRODUCT_NAME = "OpenVINO Model Server"
-PRODUCT_VERSION ?= "2023.0.0"
+PRODUCT_VERSION ?= "2023.1.0"
+PROJECT_VER_PATCH =
+
+$(eval PROJECT_VER_PATCH:=`git rev-parse --short HEAD`)
+$(eval PROJECT_NAME:=${PRODUCT_NAME})
+$(eval PROJECT_VERSION:=${PRODUCT_VERSION}.${PROJECT_VER_PATCH})
 
 OVMS_CPP_CONTAINER_NAME ?= server-test$(shell date +%Y-%m-%d-%H.%M.%S)
 OVMS_CPP_CONTAINER_PORT ?= 9178
 
 TEST_PATH ?= tests/functional/
 
-BUILD_CUSTOM_NODES ?= true
+BUILD_CUSTOM_NODES ?= false
+
+BUILD_ARGS = --build-arg http_proxy=$(HTTP_PROXY)\
+	--build-arg https_proxy=$(HTTPS_PROXY)\
+	--build-arg no_proxy=$(NO_PROXY)\
+	--build-arg ov_source_branch=$(OV_SOURCE_BRANCH)\
+	--build-arg ov_source_org=$(OV_SOURCE_ORG)\
+	--build-arg ov_contrib_org=$(OV_CONTRIB_ORG)\
+	--build-arg ov_use_binary=$(OV_USE_BINARY)\
+	--build-arg sentencepiece=$(SENTENCEPIECE)\
+	--build-arg DLDT_PACKAGE_URL=$(DLDT_PACKAGE_URL)\
+	--build-arg CHECK_COVERAGE=$(CHECK_COVERAGE)\
+	--build-arg RUN_TESTS=$(RUN_TESTS)\
+	--build-arg FUZZER_BUILD=$(FUZZER_BUILD)\
+	--build-arg build_type=$(BAZEL_BUILD_TYPE)\
+	--build-arg debug_bazel_flags=$(BAZEL_DEBUG_FLAGS)\
+	--build-arg CMAKE_BUILD_TYPE=$(CMAKE_BUILD_TYPE)\
+	--build-arg minitrace_flags=$(MINITRACE_FLAGS)\
```

Comment:
I dont see it being used in any dockerfile. Are you sure minitrace still works? There is test for that in ovms-test.